### ISSUE TITLE
REST API: Collections added to Spaces are not listed

### ISSUE THREAD

**Issue by cpudney** [clowder-framework/clowder#192]
**Describe the bug**
If I create a collection _C_ then _C_ is listed in the set of collections (GET /collections). If I then create a space _S_ and add _C_ to _S_ then _C_ is no longer listed in the set of collections

All operations relate to the REST API v1.14.1#425

**To Reproduce**
```
# Create collection
curl -X POST "https://example.com/clowder/api/collections" -H "accept: */*" -H "X-API-Key: xxxx" -H "Content-Type: application/json" -d "{\"name\":\"foo\",\"description\":\"bar\"}"
{
  "id": "604b114fe4b0bd3ebb7b2d9d"
}

# List collections matching "foo"
curl -X GET "https://example.com/clowder/api/collections?title=foo" -H "accept: */*" -H "X-API-Key: xxxx"
[
  {
    "id": "604b114fe4b0bd3ebb7b2d9d",
    "collectionname": "foo",
    "description": "bar",
    "created": "Fri Mar 12 06:59:27 UTC 2021",
    "thumbnail": null,
    "authorId": "5be1498950bbf54f6c889631",
    "resource_type": "collection"
  }
]

# Create a new space.
curl -X POST "https://example.com/clowder/api/spaces" -H "accept: */*" -H "X-API-Key: xxxx" -H "Content-Type: application/json" -d "{\"name\":\"test space\",\"description\":\"hello world!\"}"
{
  "id": "604b11cde4b0b65b434b39b8"
}

# Add collection to the space
curl -X POST "https://example.com/clowder/api/spaces/604b11cde4b0b65b434b39b8/addCollectionToSpace/604b114fe4b0bd3ebb7b2d9d" -H "accept: */*" -H "X-API-Key: xxxx"
{
  "collectionInSpace": "1"
}

# List collections matching "foo"
curl -X GET "https://example.com/clowder/api/collections?title=foo" -H "accept: */*" -H "X-API-Key: xxxx"
[]
```

**Expected behavior**
Expect the same reponse from GET /collections regardless of whether a collection is added to a space.

**Comment by tcnichol** [comment]
I have tried to replicate this but I don't get the same error. It seems to be working properly. 

@lmarini could this have possibly been fixed along with this bug?

https://github.com/clowder-framework/clowder/issues/179

Let me know if you can replicate it.

**Comment by lmarini** [comment]
It could be related to the config settings.

**Comment by cpudney** [comment]
Can  confirm this is fixed on 1.16.0 (tested on SIMPL).

**Comment by lmarini** [comment]
Thanks @cpudney !
